Material_TimePicker/src/main/ets/components/TImeModel.ets

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ClockPeriod, TimeFormat } from "./Types"
+
+export class TimeModel {
+  hour: number
+  minute: number
+  period: ClockPeriod
+  format: TimeFormat
+  constructor(hour: number, minute: number, format: TimeFormat) {
+    this.format = format
+    this.setHour(hour)
+    this.setMinute(minute)
+    this.setHourOfDay(hour)
+  }
+
+  public setHour(hour: number) {
+    if (hour < 0 || hour > 24)  return
```

Comment:
handle null & undefined validation in all setter methods

Reply:
done